syntax = "proto3";

import public "request.proto";
import public "notification.proto";

message N0stackMessage {
  string request_id = 1;
  uint32 version = 2;

  oneof message {
    Request Request = 10;
    Notification Notification = 11;
  }
}
